awscli/customizations/wizard/wizards/lambda/new-function.yml

```diff
@@ -1,12 +1,14 @@
-version: "0.1"
+version: "0.2"
```

Comment:
We should add a changelog entry for this similar to what we did for the new table wizard for DynamoDB.